recipe/meta.yaml

```diff
@@ -20,19 +20,21 @@ source:
     - osx.patch  # [osx]
 
 build:
-  number: 0
+  number: 1
   detect_binary_files_with_prefix: true
   skip: true  # [win]
 
 requirements:
   build:
     - gcc  # [osx]
+    - libgcc  # [osx]
```

Comment:
Go for it then. But maybe @msarahan can chime in. Maybe the ``libgcc`` has something different that is not in the `gcc` packages that I am not aware of.